doc/api_entitymanager.md

```diff
@@ -24,13 +24,17 @@ A `Repository` instance.
 ```js
 import {inject}        from 'aurelia-framework';
 import {EntityManager} from 'aurelia-orm';
-import UserEntity      from 'entity/user';
+import {UserEntity}    from 'entity/user';
 
 @inject(EntityManager)
 export class SomeUpdateViewModel {
   constructor (entityManager) {
     // Return the repository configured for user
     entityManager.getRepository('user');
+    
+    // Return the repository by Entity class
+    // That only works for Entities, that have the @repository() decorator
```

Comment:
This isn't true. This works for all entities. It just returns the default repository.